SecretManager/src/V1/DeleteSecretRequest.php

```diff
@@ -30,6 +30,19 @@ class DeleteSecretRequest extends \Google\Protobuf\Internal\Message
      * Generated from protobuf field <code>string etag = 2 [(.google.api.field_behavior) = OPTIONAL];</code>
      */
     private $etag = '';
+    /**
```

Comment:
nit but looks like either our whitespace or our fragment injection is one line off (there's two lines of whitespace at the bottom and one line of whitespace missing at the top)

Reply:
Hmm I think in the fragment generator we should put the space on top

Reply:
Ok, just need to prepend a new line to the "formatted" snippet content. I've got a fix prepped

Reply:
Resolving as I've pushed the version with the local fix.